packages/rpc-methods/src/restricted/confirm.ts

```diff
@@ -10,13 +10,14 @@ const methodName = 'snap_confirm';
 
 export type ConfirmMethodHooks = {
   /**
-   *
+   * @param snapId - The ID of the Snap that created the confirmation.
```

Comment:
Is a "snapId" the same thing as an "origin"?

Reply:
Ah, yes. We need some terminology cleanup to happen after launch. In RPC-method-land, `origin` and `snapId` are used interchangeably. Probably we'll move to `subjectIdentifier` or something like that?

Reply:
What will the snap ID be? Could it be expressed as an origin?

Reply:
We could perhaps use an origin, but in #140 I decided to encode the publishing location of the Snap in the ID itself to make it easy to do stuff like this: https://github.com/MetaMask/snaps-skunkworks/blob/snap-publishing-v0.1/packages/controllers/src/snaps/SnapController.ts/#L932-L939